blurhash.cpp

```diff
@@ -11,7 +11,7 @@
 #endif
 
 #ifdef DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
-#include <doctest.h>
+#include <doctest/doctest.h>
```

Comment:
This actually does not seem to work. The upstream meson file ( https://github.com/onqtam/doctest/blob/master/meson.build ), sets the include folder to doctest, which means you can't find the doctest.h, if you use the subproject. Are you sure it doesn't just work on your system, because it looks in /usr/include? I'm going to check what the installed version of doctest does now, if I can, but I think it sets the include path similarly. Otherwise we'll need to do some __has_include hackery.

Reply:
Okay, verifying that it works with a globally installed doctest will be a bit more effort for me, maybe we should do the following in the meantime?

```suggestion
#if __has_include(<doctest.h>)
#include <doctest.h>
#else
#include <doctest/doctest.h>
#endif
```

Reply:
If it's fine for you to have this hack it's fine, else I'll patch this file in guix build so that you have more time to address this in the "right" way

Reply:
We have this hack for lmdbxx in Nheko already. Sadly for single header libs their uninstalled version often slightly differs from the installed version... Thank you!